### PR TITLE
Create PRS13990Semon

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
This person with the name Səmʿon, was inscribed many times in BMLor403 (always erased). This manuscript was estimated on palaeographic grounds to the 15th century by Paolo Marrassini.

### CODE REVIEW

File: new/PRS13990Semon.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">Səmʿon</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ስምዖን</persName>
+                    <birth/>
+                    <death/>
+                    <floruit>15th century</floruit>
```
Comment (thea-m) [inline new/PRS13990Semon.xml:52]: could you include the date also for filtering purposes as values for attributes notBefore and notAfter?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you, just minimal remarks

File: new/PRS13990Semon.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/PRS13990Semon.xml:52]: Again I remember similar shortcomings in previous records - even more thank you for adressing it!

**Review by thea-m — APPROVED** [review]
No worries, I notice them because I did them myself a few hundred times :)

**Pull request merged**